ntbs-service/Services/EnhancedSurveillanceAlertsService.cs

```diff
@@ -21,7 +21,7 @@ public EnhancedSurveillanceAlertsService(IAlertService alertService)
 
         public async Task CreateOrDismissMdrAlert(INotificationForDrugResistanceImport notification)
         {
-            if (notification.IsMdr)
+            if (notification.IsMdr && !notification.MDRDetails.MDRDetailsEntered)
             {
                 await CreateMdrAlert(notification);
```

Comment:
So we're sticking with the behaviour of not reopening an alert if the setting has been toggled? What was the rationale for this?

Reply:
I discussed this for a while with Nancy. In short: it should be really quite unlikely the alert would be triggered twice anyway. The odds of entering in an MDR treatment plan by mistake is really quite low, because if the notification has not been formally submitted then the MDR questionnaire will need to be filled out before this happens. It also seems quite unlikely that a user would edit a pre-existing notification to MDR without this being necessary. It didn't feel worth breaking the pattern that alerts are triggered at most once given that the likelihood that this would be needed is so low to begin with

Reply:
Ok, that's good enough for me